db/dbHelper.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable class-methods-use-this */
+
+import { Parties } from './db';
+
+class DBhelper {
+  createParty(req, res) {
+    if (!req.body.name) {
+      return res.status(401).send({
+        status: 401,
+        error: 'name not specified',
+      });
+    }
+    if (!req.body.hqAddress) {
+      return res.status(404).send({
+        status: 401,
+        error: 'headquarters not specified',
+      });
+    }
+    if (!req.body.logoUrl) {
+      return res.status(401).send({
+        status: 401,
+        error: 'logo not specified',
+      });
+    }
+
+    // Initialize the mock parties
+    const parties = Parties;
+    for (let i = 0; i < parties.length; i += 1) {
+      if (parties[i].name === req.body.name) {
+        return res.status(401).send({
+          status: 401,
```

Comment:
Please update the status code with the correct one. You can check this site for more clarity on the status code use `https://httpstatuses.com/`